docs/site/LoopBack-3.x.md

```diff
@@ -37,80 +38,90 @@ In the persistence layer, users can contribute the following artifacts:
+LoopBack 4 is moving to the well-known Model-(View-)Controller pattern,
+where the code responsible for data access and manipulation is separated
+from the code responsible for implementing the REST API.
 
-[loopback-next-example](https://github.com/strongloop/loopback-next-example) demonstrates this loose coupling. Facade is the top-level service that serves the account summary API, and is dependent on the three services Account, Customer, and Transaction. But the facade only aggregates the calls to the three services, and is not tightly coupled with the service implementation; that's why it is independent of the three services. We can define the APIs in facade the way we want. Thus, code responsible for data access and manipulation is separated from the code responsible for implementing client side APIs.
+[loopback4-example-microservices](https://github.com/strongloop/loopback4-example-microservices)
+demonstrates this loose coupling. Facade is the top-level service that serves
+the account summary API, and is dependent on the three services Account,
+Customer, and Transaction. But the facade only aggregates the calls to the
+three services, and is not tightly coupled with the service implementation;
+that's why it is independent of the three services. We can define the APIs in
+facade the way we want. Thus, code responsible for data access and manipulation
+is separated from the code responsible for implementing client side APIs.
 
 
 | Concept/Feature       | LoopBack 3.x                                   | LoopBack 4                                        |
 | --------------------- | ---------------------------------------------- | ------------------------------------------------- |
-| Programming Language  | Built with JavaScript ES5<br>Node.js callback  | TypeScript 2.5.x & JavaScript ES2016/2017<br>Promise & Async/Await         |
-| Core foundation       | Express with LoopBack extensions               | Home-grown IoC container                           |
-| Model Definition      | Models can be defined with JavaScript or json  | Models can be defined with TypeScript/JavaScript/JSON           |
-| Model Persistence     | A model can be attached to a datasource backed by a connector that implements CRUD operations | Repository APIs are introduced to represent persistence related operations. Repository is the binding of model metadata to a datasource |
+| Programming Language  | Built with JavaScript ES5<br>Node.js callback  | TypeScript 2.6.x & JavaScript ES2016/2017<br>Promise & Async/Await |
+| Core foundation       | Express with LoopBack extensions               | Home-grown IoC container                          |
+| Model Definition      | Models can be defined with JavaScript or JSON  | Models can be defined with TypeScript/JavaScript/JSON(TBA) |
```

Comment:
Why the TBA?

Reply:
I've put TBA there because I thought we currently don't support creating a `ModelDefinition` through a JSON file.